packages/grafana-ui/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -35,13 +35,12 @@ export const colorPickerFactory = <T extends ColorPickerProps>(
     pickerTriggerRef = createRef<any>();
 
     render() {
-      const { theme, children, onChange } = this.props;
+      const { theme, children, onChange, color } = this.props;
       const styles = getStyles(theme);
       const popoverElement = React.createElement(popover, {
         ...{ ...this.props, children: null },
         onChange,
       });
-
```

Comment:
Can readd the space here

Reply:
I checked and there isn't a white space in my code. I made the updates suggested earlier and will push this again now.